alphafold3_pytorch/alphafold3.py

```diff
@@ -1984,32 +1984,38 @@ class WeightedRigidAlign(Module):
     @typecheck
     def forward(
         self,
-        pred_coords: Float['b n 3'],       # predicted coordinates
-        true_coords: Float['b n 3'],       # true coordinates
-        weights: Float['b n'],             # weights for each atom
-        mask: Bool['b n'] | None = None    # mask for variable lengths
+        true_coords: Float['b n 3'],        # true coordinates
```

Comment:
Equation 28 of the AF3 supplement appears to be entirely correct (except for the typo that should instead write `\vec{\mu}^{GT}` on Line 11). In which case, I think it would be more confusing for readers of this code to have to mentally invert all the operations they might read in the supplement when looking at this code. Since we still call this function with `true_coords` as the first argument, all operations in this function should match the AF3 paper.